northstar/src/runtime/mount.rs

```diff
@@ -52,6 +59,8 @@ pub enum Error {
     JoinError(tokio::task::JoinError),
     #[error("Os error: {0:?}")]
     Os(nix::Error),
+    #[error("Repository error: {0:?}")]
+    Repository(String),
```

Comment:
What is a repository error? Maybe this is more a "InvalidRepository" or "UnknownRepository"

Reply:
We only use it in case there is "No public key in repository to verify NPK signature". Better rename it to 'MissingKey' error or something like that.